code/game/objects/structures/inflatable.dm

```diff
@@ -163,6 +166,9 @@
 		return !opacity
 	return !density
 
+/obj/structure/inflatable/door/CanAtmosPass(turf/T)
```

Comment:
I kept having water flowing straight through my inflatables when I tried to make airlocks, so almost certainly not (or broken if it is).